src/api/app/views/webui/cloud/upload_jobs/new.html.haml

```diff
@@ -21,6 +21,9 @@
   %p
     = upload_job_form.label 'cloud_backend_upload_job[region]', "Region:"
     = select_tag('cloud_backend_upload_job[region]', options_for_select(@ec2_regions))
+  %p
+    = upload_job_form.label 'cloud_backend_upload_job[vpc_subnet_id]', 'VPC Subnet ID (optional):'
```

Comment:
Maybe we should add a 'hint' text here that if the user does not provide the vpc subnet id, OBS will create it.

cc @bgeuken @evanrolfe